<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ApplyExchangeInfoDialog</class>
 <widget class="QDialog" name="ApplyExchangeInfoDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>960</width>
    <height>543</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>960</width>
     <height>543</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft YaHei UI Light</family>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <widget class="QWidget" name="containerWidget" native="true">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>110</y>
      <width>460</width>
      <height>320</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>20</y>
       <width>200</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>微软雅黑</family>
       <pointsize>-1</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel{font:  14px &quot;微软雅黑&quot;;color: rgb(51, 51, 51);}

</string>
     </property>
     <property name="text">
      <string>提交上币申请信息</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QToolButton" name="closeBtn">
     <property name="geometry">
      <rect>
       <x>447</x>
       <y>5</y>
       <width>7</width>
       <height>7</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>70</y>
       <width>80</width>
       <height>24</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel{font: 11px &quot;微软雅黑&quot;;color: rgb(137, 129, 161);}
</string>
     </property>
     <property name="text">
      <string>选择合约</string>
     </property>
    </widget>
    <widget class="QToolButton" name="okBtn">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>260</y>
       <width>96</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>微软雅黑</family>
       <pointsize>10</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>提交申请</string>
     </property>
    </widget>
    <widget class="QToolButton" name="cancelBtn">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>260</y>
       <width>96</width>
       <height>30</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>取消</string>
     </property>
    </widget>
    <widget class="QLabel" name="tipLabel">
     <property name="geometry">
      <rect>
       <x>45</x>
       <y>110</y>
       <width>261</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel{font: 11px &quot;微软雅黑&quot;;color: rgb(255,0,0);}
</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QComboBox" name="contractComboBox">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>70</y>
       <width>300</width>
       <height>24</height>
      </rect>
     </property>
    </widget>
    <widget class="QComboBox" name="accountComboBox">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>110</y>
       <width>120</width>
       <height>24</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>110</y>
       <width>80</width>
       <height>24</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel{font: 11px &quot;微软雅黑&quot;;color: rgb(137, 129, 161);}
</string>
     </property>
     <property name="text">
      <string>选择申请账户</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>150</y>
       <width>80</width>
       <height>24</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel{font: 11px &quot;微软雅黑&quot;;color: rgb(137, 129, 161);}
</string>
     </property>
     <property name="text">
      <string>联系方式</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="phoneLineEdit">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>150</y>
       <width>300</width>
       <height>24</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
